07/27/2024 03:26:42 PM Logging starts
Traceback (most recent call last):
  File "./popular_projects_snippets_dataset/black/bodies/body_162_guide_2_0.py", line 15, in <module>
    Callable[[Sequence[Leaf]], Iterator[Tuple[Index, Leaf]]],
NameError: name 'Leaf' is not defined
Command exited with non-zero status 1
1.09user 2.64system 0:00.72elapsed 512%CPU (0avgtext+0avgdata 104920maxresident)k
0inputs+16outputs (1major+23983minor)pagefaults 0swaps
